src/briefcase/integrations/android_sdk.py

```diff
@@ -47,6 +47,10 @@ class AndroidSDK(ManagedTool):
     name = "android_sdk"
     full_name = "Android SDK"
 
+    # Latest version for Command-Line Tools download as of August 2023
```

Comment:
Would be helpful add a note reminding whoever updates this to also update the documentation reference in android.rst, so that the two don't get out of alignment.

Reply:
Good call; added a note.